og_ui/og_ui.module

```diff
@@ -8,6 +8,7 @@
 use Drupal\Core\Config\Entity\ConfigEntityBundleBase;
 use Drupal\Core\Entity\BundleEntityFormBase;
 use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityTypeInterface;
```

Comment:
This is the only change in this file. It seems to be unused, but it is strange that PHP_CodeSniffer did not pick up on this.